internal/controller/kyma/setup.go

```diff
@@ -61,6 +61,7 @@ func (r *Reconciler) SetupWithManager(mgr ctrl.Manager, opts ctrlruntime.Options
 	if err := ctrl.NewControllerManagedBy(mgr).For(&v1beta2.Kyma{}).
 		Named(controllerName).
 		WithOptions(opts).
+		WithEventFilter(predicates).
```

Comment:
Since we all agreed on adding changes to the respective _test.go file, do you plan on skipping it for this one?